src/plugins/plugin.legend.js

```diff
@@ -104,6 +104,10 @@ var Legend = Element.extend({
 	initialize: function(config) {
 		helpers.extend(this, config);
 
+		if (this.options && !this.options.align) {
```

Comment:
A lot of `this` here, should add a `var me=this;`

Reply:
A good rule is once `this` is used 4 times, it saves bytes to capture it locally

Reply:
👍